torch/__future__.py

```diff
@@ -1,21 +1,60 @@
-"""
-This global flag controls whether to assign new tensors to the parameters
-instead of changing the existing parameters in-place when converting an `nn.Module`
-using the following methods:
-1. `module.cuda()` / `.cpu()` (for moving `module` between devices)
-2. `module.float()` / `.double()` / `.half()` (for converting `module` to a different dtype)
-3. `module.to()` / `.type()` (for changing `module`'s device or dtype)
-4. `module._apply(fn)` (for generic functions applied to `module`)
+_overwrite_module_params_on_conversion: bool = False
+_swap_module_params_on_conversion: bool = False
 
-Default: False
-"""
-_overwrite_module_params_on_conversion = False
 
+def set_overwrite_module_params_on_conversion(value: bool) -> None:
+    """
+    Sets whether to  assign new tensors to the parameters instead of changing the
+    existing parameters in-place when converting an ``nn.Module``.
 
-def set_overwrite_module_params_on_conversion(value):
+    When enabled, the following methods will assign new parameters to the module:
+
+    #. ``module.{device}()`` (e.g. ``module.cuda()``) for moving a module between devices
+    #. ``module.{dtype}()`` (e.g. ``module.float()``) for converting a module to a different dtype
+       (for converting a module to a different dtype)
+    #. ``module.to()``
```

Comment:
should we mention `_apply()`? I omitted it since it is a private method

Reply:
No we shouldn't talk about it.